module/mempool/consensus/exec_fork_suppressor.go

```diff
@@ -337,41 +343,6 @@ func hasConsistentStateTransitions(irSeal, irSeal2 *flow.IncorporatedResultSeal)
 	return true
 }
 
-// checkExecutionForkDetected checks the database whether evidence
-// about an execution fork is stored. Returns the stored evidence.
-func checkExecutionForkEvidence(db *badger.DB) ([]*flow.IncorporatedResultSeal, error) {
```

Comment:
`checkExecutionForkEvidence()` and `storeExecutionForkEvidence()` are implemented in `storage/store/execution_fork_evidence.go`, so database related functionality is in `/storage`.